app/scripts/vendor/env-test.js

```diff
@@ -31,13 +31,8 @@
   var pwElement = document.createElement('input');
   pwElement.setAttribute('type', 'password');
 
-  try {
-    // IE8 will blow up here.
-    pwElement.setAttribute('type', 'text');
-    docElement.className += ' toggle-pw';
-  } catch(e) {
-    docElement.className += ' no-toggle-pw';
-  }
+  pwElement.setAttribute('type', 'text');
+  docElement.className += ' toggle-pw';
```

Comment:
@riadhchtara, @vladikoff - I tried figuring out a way to query the browser to see if it supports a show/hide, but had no luck. I just may have not looked hard enough.

If we remove `no-toggle-pw`, `toggle-pw` is unnecessary. `toggle-pw` is unused in both the CSS and JS.